flake8 3.2 reports new errors

Looks like the recently released 3.2 version of flake8 finds a few more errors than the previous version:

```
$ flake8 pouta_blueprints
pouta_blueprints/app.py:64:1: E305 expected 2 blank lines after class or function definition, found 1
pouta_blueprints/drivers/provisioning/pvc_cmdline_driver.py:269:1: E305 expected 2 blank lines after class or function definition, found 1
pouta_blueprints/tests/test_flask_api.py:1150:1: E305 expected 2 blank lines after class or function definition, found 0
pouta_blueprints/tests/selenium_test_login.py:135:1: E305 expected 2 blank lines after class or function definition, found 1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,7 +8,7 @@ sudo: false
 
 install:
 - pip install -r requirements.txt
-- pip install flake8 --use-mirrors
+- pip install flake8==3.2.0
 - pip install .
 
 before_script:

--- a/pouta_blueprints/app.py
+++ b/pouta_blueprints/app.py
@@ -61,4 +61,5 @@ def run_things_in_context(test_run):
         if not test_run:
             Variable.sync_local_config_to_db(BaseConfig, app.dynamic_config)
 
+
 run_things_in_context(test_run)

--- a/pouta_blueprints/drivers/provisioning/pvc_cmdline_driver.py
+++ b/pouta_blueprints/drivers/provisioning/pvc_cmdline_driver.py
@@ -265,6 +265,7 @@ class PvcCmdLineDriver(base_driver.ProvisioningDriverBase):
     def do_housekeep(self, token):
         pass
 
+
 # testing templating
 if __name__ == '__main__':
     blueprint_config = {

--- a/pouta_blueprints/tests/selenium_test_login.py
+++ b/pouta_blueprints/tests/selenium_test_login.py
@@ -132,5 +132,6 @@ class LoginTestCase(SeleniumBaseTestCase):
         shibboleth_enabled.value = saved
         self.db.session.commit()
 
+
 if __name__ == "__main__":
     unittest.main()

--- a/pouta_blueprints/tests/test_flask_api.py
+++ b/pouta_blueprints/tests/test_flask_api.py
@@ -1147,5 +1147,7 @@ class FlaskApiTestCase(BaseTestCase):
             method='GET',
             path='/api/v1/stats')
         self.assertStatus(response, 403)
+
+
 if __name__ == '__main__':
     unittest.main()

